ros/lib/models.py

```diff
@@ -28,6 +32,25 @@ def idling_time(self):
             return "%0.2f" % idling_percent
 
 
+class PerformanceProfileHistory(db.Model):
+    performance_record = db.Column(JSONB)
+    performance_utilization = db.Column(JSONB)
+    report_date = db.Column(
+        db.DateTime(timezone=True), default=datetime.datetime.utcnow)
```

Comment:
@kgaikwad Shall we remove the `default` from here, since profiles are supposed to copied into this table?

Reply:
It is not actually run copy command here but manual insertion of records in both tables.
Though currently it is using same fields data while inserting records, I will still prefer to keep this `default` at model definition level. 
Any thoughts?

Reply:
Also, this way if a record shows up  in performance_profile_history without report_date, we'll know something is fishy

Reply:
In case we need it, I'm all for it, but since `performance_profile_history` will essentially retain copies, we can expect `report_date` in every respective record, just a thought, thanks!